src/haddock/modules/refinement/openmm/defaults.yaml

```diff
@@ -0,0 +1,206 @@
+  long: Indicate use of implicit solvent, avoids building a water solvation box around pdbs. 
+  group: ''
+  explevel: easy
+
+implicit_solvent_model:
+  default: 'implicit/hct.xml'
+  type: string
+  minchars: 0
+  maxchars: 100
+  choices:
+    - 'implicit/hct.xml'
+    - 'implicit/obc1.xml'
+    - 'implicit/obc2.xml'
+    - 'implicit/gbn.xml'
+    - 'implicit/gbn2.xml'
+  title: Implicit solvent models.
+  short: Implicit solvent models supported by OpenMM.
+  long: Implicit solvent models supported by OpenMM.
+  group: ''
+  explevel: easy
+
+add_extra_particles_for_forcefield:
+  default: false
+  type: boolean
+  title: Add extra particles for forcefield.
+  short: Add extra particles for forcefield if needed such as e.g. Drude particles.
+  long: Add extra particles for forcefield if needed such as e.g. Drude particles.
+  group: ''
+  explevel: easy
+
+timestep_ps:
+  default: 0.004
+  type: float
+  precision: 3
+  min: 0
+  max: 0.1
+  title: Timestep definition.
+  short: Definition of timestep used by the integrator to progress the simulation.
+  long: Definition of timestep used by the integrator to progress the simulation.
+  group: ''
+  explevel: easy
+
+constraints:
+  default: 'HBonds'
+  type: string
+  minchars: 0
+  maxchars: 100
+  choices:
+    - 'None'
+    - 'HBonds'
+    - 'AllBonds'
+    - 'HAngles'
+  title: Integration constraints.
+  short: Integration constraints used to allow for larger integration time step.
+  long: Integration constraints used to allow for larger integration time step. For extra info see http://docs.openmm.org/latest/userguide/application/02_running_sims.html#constraints
+  group: ''
+  explevel: easy
+
+equilibration_timesteps:
+  default: 1000
+  type: integer
+  min: 0
+  max: 100000
+  title: Equilibration timesteps.
+  short: Equilibration timesteps performed without saving the simulation state.
+  long: Equilibration timesteps performed without saving the simulation state.
+  group: ''
+  explevel: easy
+
+simulation_timesteps:
+  default: 1000
+  type: integer
+  min: 1
+  max: 1000000
+  title: Simulation timesteps.
+  short: Simulation timesteps performed during the simulation.
+  long: Simulation timesteps performed during the simulation.
+  group: ''
+  explevel: easy
+
+center_atoms:
+  default: false
+  type: boolean
+  title: Move atoms to the center of the solvation box.
+  short: Move atoms to the center of the solvation box. The pdb is placed around the origin, and water molecules of the solvation box are build up from the origin.
+  long: Move atoms to the center of the solvation box. The pdb is placed around the origin, and water molecules of the solvation box are build up from the origin. Not centering the atoms shouldn't be a problem if periodic boundary conditions are deployed, but employing it also prevents the molecule from jumping around the boundary while visually inspecting the simulation in e.g. PyMOL.
```

Comment:
FYI:

It is possible to split into multiple lines as:

```yaml
  long: Move atoms to the center of the solvation box.
    The pdb is placed around the origin, and water molecules of the solvation box are build up from the origin.
    Not centering the atoms shouldn't be a problem if periodic boundary conditions are deployed,
    but employing it also prevents the molecule from jumping around the boundary while visually
    inspecting the simulation in e.g. PyMOL.
```

Reply:
oh nice, will do!